README.md

```diff
@@ -8,9 +8,9 @@ This repository contains several .NET libraries (originally developed as part of
-[`CommunityToolkit.Diagnostics`](https://learn.microsoft.com/windows/communitytoolkit/diagnostics/introduction) | [![CommunityToolkit.Diagnostics](https://img.shields.io/nuget/v/CommunityToolkit.Diagnostics)](https://nuget.org/packages/CommunityToolkit.Diagnostics/) | [![CommunityToolkit.Diagnostics](https://img.shields.io/nuget/vpre/CommunityToolkit.Diagnostics)](https://nuget.org/packages/CommunityToolkit.Diagnostics/absoluteLatest) | A set of helper APIs (specifically, [`Guard`](https://learn.microsoft.com/windows/communitytoolkit/developer-tools/guard) and [`ThrowHelper`](https://learn.microsoft.com/windows/communitytoolkit/developer-tools/throwhelper)) that can be used for cleaner, more efficient and less error-prone argument validation and error checking.
-[`CommunityToolkit.HighPerformance`](https://learn.microsoft.com/windows/communitytoolkit/high-performance/introduction) | [![CommunityToolkit.HighPerformance](https://img.shields.io/nuget/v/CommunityToolkit.HighPerformance)](https://nuget.org/packages/CommunityToolkit.HighPerformance/) | [![CommunityToolkit.HighPerformance](https://img.shields.io/nuget/vpre/CommunityToolkit.HighPerformance)](https://nuget.org/packages/CommunityToolkit.HighPerformance/absoluteLatest) | A collection of helpers for working in high-performance scenarios. It includes APIs such as [pooled buffer helpers](https://learn.microsoft.com/windows/communitytoolkit/high-performance/memoryowner), a fast [string pool](https://learn.microsoft.com/windows/communitytoolkit/high-performance/stringpool) type, a 2D variant of `Memory<T>` and `Span<T>` ([`Memory2D<T>`](https://learn.microsoft.com/windows/communitytoolkit/high-performance/memory2d) and [`Span2D<T>`](https://learn.microsoft.com/windows/communitytoolkit/high-performance/span2d)) also supporting discontiguous regions, helpers for bit shift operations (such as [`BitHelper`](https://learn.microsoft.com/windows/communitytoolkit/high-performance/span2d), also used in [Paint.NET](https://getpaint.net)), and more.
+[`CommunityToolkit.Common`](https://learn.microsoft.com/dotnet/api/?term=communitytoolkit.common) | [![CommunityToolkit.Common](https://img.shields.io/nuget/v/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/) | [![CommunityToolkit.Common](https://img.shields.io/nuget/vpre/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/absoluteLatest) | A set of helper APIs shared with other [`CommunityToolkit`](https://learn.microsoft.com/dotnet/communitytoolkit/) libraries.
```

Comment:
The first link https://learn.microsoft.com/dotnet/api/?term=communitytoolkit.common
points to a search page result for now, but https://learn.microsoft.com/dotnet/api/communitytoolkit.common seems to get to the actual api page. However, since the version 8, no real automatic complete api docs were available, always referring to the old docs, which don't include the newest source generators or other overload with good docs in the source files. Even more, up until recently, the old hand written docs for the new community toolkit weren't published with the new usage patterns, but it seems to have been done in the last weeks

Reply:
Updated the link